lms/static/scripts/frontend_apps/index.js

```diff
@@ -21,19 +21,17 @@ import { registerIcons } from './components/SvgIcon';
 import iconSet from './icons';
 registerIcons(iconSet);
 
-let rpcServer;
-if (config.mode === 'basic-lti-launch') {
-  // Create an RPC Server and start listening to postMessage calls.
-  rpcServer = startRpcServer({
-    allowedOrigins: config.rpcServer.allowedOrigins,
-    clientConfig: config.hypothesisClient,
-  });
-}
-
 render(
   <Config.Provider value={config}>
     {config.mode === 'basic-lti-launch' && (
-      <BasicLtiLaunchApp rpcServer={rpcServer} />
+      <BasicLtiLaunchApp
+        clientRpc={
```

Comment:
This might be a _touch_ easier to parse if the `ClientRpc` were created outside of `render`, but I suppose it's mostly stylistic preference...

Reply:
The `config` properties needed to instantiate `ClientRpc` are only set if `config.mode === 'basic-lti-launch'`, so there is a small benefit in that this avoids the need for two separate `if (config.mode === 'basic-lti-launch')` blocks.